src/main/java/com/thebois/views/toolbar/BuildMenuToolbarView.java

```diff
@@ -32,10 +33,7 @@ private Actor createStructureButtonContainer(
         final Skin skin, final IEventListener<OnClickEvent<StructureType>> listener) {
 
         final Table toolbarTable = createTable();
-
-        final ButtonGroup<StructureButton> buttonGroup = createButtonGroup();
-
-        createStructureButtons(skin, listener, toolbarTable, buttonGroup);
+        createStructureButtons(skin, listener, toolbarTable, structureButtonGroup);
```

Comment:
No need to pass `structureButtonGroup` as an argument when it's a field in the class.

Reply:
👍